### ISSUE TITLE
Exact not working for subItems

### ISSUE THREAD

**Issue by NickAlvesX** [tabler/tabler-react#407]
I see that the merge #383 solve the _exact_ question on the nav items, but there's none such implementation for the navBar subItems.

For example:

```
{
    value: 'Orders',
    icon: 'package',
    subItems: [
        {value: 'New order', to: '/orders/new', LinkComponent: withRouter(NavLink)},
        {value: 'Orders', to: '/orders', LinkComponent: withRouter(NavLink)}
    ]
}
```

Would it be nice to `useExact: true` on each subItem.

**Comment by Thomas-Smyth** [comment]
I can look into this either today or tomorrow if no one else looks into it first.

**Comment by AaronCoplan** [comment]
@Thomas-Smyth Feel free to look into this!  Thanks!

**Comment by jonthomp** [comment]
This has been released in 1.30.1
